diff --git a/node_modules/expo-modules-core/common/cpp/fabric/ExpoViewComponentDescriptor.cpp b/node_modules/expo-modules-core/common/cpp/fabric/ExpoViewComponentDescriptor.cpp
index dbfca2c..775cd77 100644
--- a/node_modules/expo-modules-core/common/cpp/fabric/ExpoViewComponentDescriptor.cpp
+++ b/node_modules/expo-modules-core/common/cpp/fabric/ExpoViewComponentDescriptor.cpp
@@ -56,12 +56,12 @@ void ExpoViewComponentDescriptor::adopt(facebook::react::ShadowNode &shadowNode)
     bool changedStyle = false;
     
     if (!isnan(styleWidth)) {
-      style.setDimension(facebook::yoga::Dimension::Width, facebook::yoga::StyleSizeLength::points(styleWidth));
+      style.setDimension(facebook::yoga::Dimension::Width, facebook::yoga::StyleLength::points(styleWidth));
       changedStyle = true;
     }
     
     if (!isnan(styleHeight)) {
-      style.setDimension(facebook::yoga::Dimension::Height, facebook::yoga::StyleSizeLength::points(styleHeight));
+      style.setDimension(facebook::yoga::Dimension::Height, facebook::yoga::StyleLength::points(styleHeight));
       changedStyle = true;
     }
     
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm
index b33b454..e02521b 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm
@@ -35,8 +35,8 @@
   if (utf8 != nullptr && expo::isAllASCIIAndNotNull(utf8, utf8 + length)) {
     return jsi::String::createFromAscii(runtime, (const char *)utf8, length);
   }
-  // Using cStringUsingEncoding should be fine as long as we provide the length.
-  return jsi::String::createFromUtf16(runtime, (const char16_t *)[value cStringUsingEncoding:NSUTF16StringEncoding], length);
+  // Using UTF8String as createFromUtf16 is not available in this JSI version.
+  return jsi::String::createFromUtf8(runtime, [value UTF8String]);
 #else
   // TODO(@jakex7): Remove after update to react-native-macos@0.79.0
   return jsi::String::createFromUtf8(runtime, [value UTF8String]);
